python/hail/ir/table_ir.py

```diff
@@ -203,3 +203,13 @@ def __init__(self, child):
 
     def __str__(self):
         return f'(TableDistinct {self.child})'
+
+class TableRepartition(TableIR):
+    def __init__(self, child, n, shuffle):
+        super().__init__()
+        self.child = child
+        self.n = n
+        self.shuffle = shuffle
+
+    def __str__(self):
+        return f'(TableRepartition {self.n} {self.shuffle} {self.child})'
```

Comment:
wondering why the order is switched from above. do we have a convention on argument order, e.g. child goes last?

Reply:
this is a result of the way `Pretty()` is written in Scala -- IR children are automatically written out, but you can optionally write a list of other parse requirements, which are emitted first.

```
            case TableRepartition(_, n, shuffle) => n.toString + " " + prettyBooleanLiteral(shuffle)
```